### PR TITLE
Include dv's when called DataFrame on the output of predict

### PR DESCRIPTION
**Pull request by pkofod**
This is super useful for plotting the diagnostics.
fixes #873

### CODE REVIEW

File: src/estimation/diagnostics.jl [hunk 1]
```diff
@@ -616,8 +617,9 @@ function DataFrames.DataFrame(i::FittedPumasModelInspection; include_covariates=
       end
     end
   end
-  ebes_df[!, :ebes_approx] .= summary(i.ebes.approx)
+  ebes_df[!, :ebes_approx] .= fill(summary(i.ebes.approx), size(ebes_df, 1))
```
Comment (andreasnoack) [inline src/estimation/diagnostics.jl]: Why was it necessary to add the `fill` here?

File: src/estimation/diagnostics.jl [hunk 2]
```diff
@@ -616,8 +617,9 @@ function DataFrames.DataFrame(i::FittedPumasModelInspection; include_covariates=
       end
     end
   end
-  ebes_df[!, :ebes_approx] .= summary(i.ebes.approx)
+  ebes_df[!, :ebes_approx] .= fill(summary(i.ebes.approx), size(ebes_df, 1))
   ebes_df = select!(select!(ebes_df, Not(:id)), Not(:time))
+  size(pred_df)
```
Comment (andreasnoack) [inline src/estimation/diagnostics.jl]: This one shouldn't be there, right?

Comment (pkofod) [reply src/estimation/diagnostics.jl]: nope :)

File: src/estimation/diagnostics.jl (hunk 1 above)
Comment (pkofod) [reply src/estimation/diagnostics.jl]: It tried to add the first letter to the first line, the second letter to the second line,... Maybe a `Ref` would do it?

**Pull request merged**